### PR TITLE
Add --getstarted flag for creating a protobuf file

### PR DESCRIPTION
**Pull request by hasLeland**
This PR adds the `--getstarted` flag to truss, which gives you a basic service-compatible protobuf file "out of the box". Here's what its output looks like:

```
$ truss --getstarted
INFO[0000] A "starter" protobuf file named 'getstarted.proto' has been created in the
current directory. You can generate a service based on this new protobuf file
at any time using the following command:

    truss getstarted.proto

If you want to generate a protofile with a different name, use the
'--getstarted' option with the name of your choice after '--getstarted'. For
example, to generate a 'foo.proto', use the following command:

    truss --getstarted foo
 
```

## Notes for reviewers:

In addition to the usual code review, please edit the english help messages in the `truss/getstarted/template.go` for consistency, tone, and grammar.

### CODE REVIEW

File: truss/getstarted/getstarted.go
Comment (hasAdamr) [inline truss/getstarted/getstarted.go]: I feel like the `dotProtoInName` template could be an inline constant in this function. 

I don't think it will be used anywhere else. And it is nice to read the message in context.

File: truss/getstarted/getstarted.go
Comment (hasAdamr) [inline truss/getstarted/getstarted.go]: Nit: To me, this function sounds like it will remove a `.proto` file.

File: truss/getstarted/getstarted.go
Comment (hasAdamr) [inline truss/getstarted/getstarted.go]: Possibly move rendering `nextStep` to just before you use it. Kind of out of place to have it here from my perspective.

File: truss/getstarted/getstarted.go
Comment (hasAdamr) [inline truss/getstarted/getstarted.go]: Are package names not allowed to have `_`'s?

File: truss/getstarted/getstarted.go
Comment (hasAdamr) [inline truss/getstarted/getstarted.go]: Maybe:
```
return nil, errors.Wrap(...
```

File: truss/getstarted/getstarted.go
Comment (hasAdamr) [inline truss/getstarted/getstarted.go]: `default.proto`?

**Review by hasAdamr — COMMENTED** [review]
Looking good! 

A few things I would like to hear your reasoning for and/or changed before I click the approve button.

Will be nice to have this for creating a new service. :open_hands:

File: truss/getstarted/getstarted.go
Comment (hasLeland) [reply truss/getstarted/getstarted.go]: There's nothing preventing them from having underscores; when implementing this I removed underscores due to some notion of aesthetics that was in my brain. I'll remove that if you think they should be allowed.

File: truss/getstarted/getstarted.go
Comment (hasLeland) [reply truss/getstarted/getstarted.go]: Whoops, should be changed to `getstarted.proto`; good catch.

File: truss/getstarted/getstarted.go
Comment (hasLeland) [reply truss/getstarted/getstarted.go]: Good point, changing to `removeDotProtoSuffix`

File: truss/getstarted/getstarted.go
Comment (hasAdamr) [inline truss/getstarted/getstarted.go]: Won't the file name be based on what is passed to this function?

File: truss/getstarted/getstarted.go
Comment (hasAdamr) [inline truss/getstarted/getstarted.go]: What happened to the comment about this being rendered early? It is kind of confusing by itself.

File: truss/getstarted/getstarted.go
Comment (hasAdamr) [inline truss/getstarted/getstarted.go]: `pinfo.FileName()` correct?

File: cmd/truss/main.go
Comment (hasAdamr) [inline cmd/truss/main.go:33]: Should there be some documentation about how you can pass something to this to get your own name for the file/package/servicename?

File: truss/getstarted/getstarted.go
Comment (hasAdamr) [reply truss/getstarted/getstarted.go]: Kind of odd to me. Really I don't mind either way. So I guess leave it? ¯\\\_(ツ)_/¯

**Review by hasAdamr — COMMENTED** [review]
Couple more things before that Approve checkmark.

Thanks for looking over the feedback.

File: cmd/truss/main.go
Comment (hasLeland) [reply cmd/truss/main.go:33]: There is documentation, shown when you use the option. https://github.com/TuneLab/go-truss/pull/196/files#diff-b7a48309a1d9f912b7f8857bd593df6eR38

Because the `getStartedFlag` is actually a bool flag and the name you can provide is actually handled outside of the way the flag package canonically does assignment, I don't want to say in the help message that you can provide a name. If we did, someone might think it means you can pass `--getstarted=foo` which will not work and will cause the program to fail.

So instead, I put the longer, more explicit instructions on how to do it in the `nextSteps` message printed when you actually use `--getstarted`.

**Review by hasAdamr — APPROVED** [review]
Great! :+1: 

Though wait for a review from either @zaquestion or @hasIan before merging though. Maybe ping them if you want to get this in sooner rather than later.

File: truss/getstarted/getstarted.go
Comment (zaquestion) [reply truss/getstarted/getstarted.go]: If its a legal in protobuf/go it should probably be allowed and not enforced by us.

Comment (hasLeland) [reply truss/getstarted/getstarted.go]: Alright, taking it out.

File: truss/getstarted/template.go
Comment (zaquestion) [inline truss/getstarted/template.go]: Thoughts on lower casing this? Believe that would be more standard for REST

Comment (hasLeland) [reply truss/getstarted/template.go]: It was that way originally because it was copy-pasted from a multiverse interface definition, but having it be lowercase is an improvement. Will make the change.

**Pull request merged**
